code/game/machinery/computer/card.dm

```diff
@@ -475,6 +482,27 @@ GLOBAL_VAR_INIT(time_last_changed_position, 0)
 							modify.access += access_type
 							log_id("[key_name(usr)] added [get_access_desc(access_type)] to [modify] using [scan] at [AREACOORD(usr)].")
 						playsound(src, "terminal_type", 50, 0)
+
+		if("access_give_aa")
+			if(authenticated == 2)
+				var/tried = 0
+				for(var/each in get_all_accesses())
+					if(!(each in modify.access))
+						modify.access += each
+						tried = TRUE
```

Comment:
```suggestion
				modify.access.Cut()
				modify.access += get_all_accesses()
```

Reply:
Cut() strips CC accesses too, shouldn't be here.

Reply:
Then don't do the cut, do |= instead

Reply:
these accesses are list, not binary.

Reply:
changes as |= now.